pkg/provider-local/webhook/prometheus/mutator.go

```diff
@@ -57,7 +57,7 @@ func (m *mutator) Mutate(_ context.Context, newObj, _ client.Object) error {
 				},
 				{
 					SourceLabels: []monitoringv1.LabelName{"__name__"},
-					Regex:        "(up|flow_.+)",
+					Regex:        "(up|flow_.+|shoot_.+)",
```

Comment:
Comparing with `pkg/utils/flow/metrics.go`, shouldn't this rather be `gardenlet_.+`?
```suggestion
					Regex:        "(up|flow_.+|gardenlet_.+)",
```

Reply:
I thought it should be `shoot_.+` because of the metric name.
@timebertt What do you think?

Reply:
@LucaBernstein is right, the metric names start with `gardener_shoot_`